drivewise/src/Components/Big_box/Big_box.js

```diff
@@ -0,0 +1,27 @@
+import React from "react"; 
+import './Big_box.css'
+export default function Big_box({header_text, paragraph_text, button_text, button_link, link_text, link_link, image, flip}) {
```

Comment:
make an interface like this 

interface arguments {
  header_text: string;
  paragraph_text: string;
  button_text: string;
  button_link: number;
  etc.
}

Reply:
Also change from Big_box to Box